ChangeLog

```diff
@@ -1,3 +1,75 @@
+* Thu Oct 15 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
+- Changesets: 497
+- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+
+- Features added since Pacemaker-2.0.4
+  + cibsecret: read value to set from input (or stdin) if not specified
+  + controller: add new IPC API command for getting node list
+  + libpe_status: Add type="integer" to rule elements, allowing for
+				  specifying 64-bit integers.
+  + scheduler: Add rule-based tests to op_defaults and rsc_defaults.
+  + scheduler: Add on-fail=demote and no-quorum-policy=demote recovery
+               policies for promoted resources.
+  + tools: Add --resource= to filter crm_mon output for a resource.
+  + tools: Add -P to crmadmin to ping via pacemakerd-api.
+  + tools: Use glib for cmdline handling in crmadmin and crm_resource.
+
+- Fixes for regressions introduced in Pacemaker-2.0.4
+  + scheduler: Add the node name back to bundle instances.
```

Comment:
This is user-visible only in crm_mon, so I'd move it to "tools:" and mention crm_mon (or something like "status displays" or "crm_mon and other tools" if you want to be more general to include crm_resource etc.)